tests/genericmetadata_test.py

```diff
@@ -22,6 +23,78 @@ def test_metadata_overlay(md: comicapi.genericmetadata.GenericMetadata, replaced
     assert md == expected
 
 
+def test_metadata_overlay_overwrite():
+    md = comicapi.genericmetadata.GenericMetadata(series="test", issue="1", title="test", genres={"test", "test2"})
+    overwrite = comicapi.genericmetadata.GenericMetadata(
+        series="test2", issue="2", title="test2", genres={"test3", "test4"}
+    )
+    md.overlay(overwrite, OverlayMode.overwrite)
+
+    assert md == overwrite
+
+
+def test_metadata_overlay_overwrite_no_clear():
+    md = comicapi.genericmetadata.GenericMetadata(
+        series="test", issue="1", title="test", genres={"test", "test2"}, story_arcs=["arc"]
+    )
+    overwrite = comicapi.genericmetadata.GenericMetadata(
+        series="test2", issue="2", title="test2", genres=set(), story_arcs=[]
+    )
+    expected = comicapi.genericmetadata.GenericMetadata(
+        series="test2", issue="2", title="test2", genres={"test", "test2"}, story_arcs=["arc"]
+    )
+    md.overlay(overwrite, OverlayMode.overwrite_no_clear)
+
+    assert md == expected
+
+
+def test_metadata_overlay_add_missing():
+    md = comicapi.genericmetadata.GenericMetadata(series="test", issue="1", title="test", genres={"test", "test2"})
+    add = comicapi.genericmetadata.GenericMetadata(
+        series="test2", issue="2", title="test2", genres={"test3", "test4"}, issue_count=5
+    )
+    expected = comicapi.genericmetadata.GenericMetadata(
+        series="test", issue="1", title="test", genres={"test", "test2"}, issue_count=5
+    )
+    md.overlay(add, OverlayMode.add_missing)
+
+    assert md == expected
+
+
+def test_metadata_overlay_combine_new():
+    md = comicapi.genericmetadata.GenericMetadata(
+        series="test", issue="1", title="test", genres={"test", "test2"}, story_arcs=["arc1"]
+    )
+    combine_new = comicapi.genericmetadata.GenericMetadata(
+        series="test2", title="test2", genres={"test2", "test3", "test4"}, story_arcs=["arc1", "arc2"]
+    )
+    expected = comicapi.genericmetadata.GenericMetadata(
+        series="test2",
+        issue="1",
+        title="test2",
+        genres={"test", "test2", "test3", "test4"},
+        story_arcs=["arc1", "arc2"],
+    )
+    md.overlay(combine_new, OverlayMode.combine_new)
+
+    assert md == expected
+
+
+def test_metadata_overlay_combine_cur():
+    md = comicapi.genericmetadata.GenericMetadata(
+        series="test", issue="1", title="test", genres={"test", "test2"}, story_arcs=["arc1"]
+    )
+    combine_cur = comicapi.genericmetadata.GenericMetadata(
+        series="test2", title="test2", genres={"test2", "test3", "test4"}, story_arcs=["arc1", "arc2"]
+    )
+    expected = comicapi.genericmetadata.GenericMetadata(
+        series="test", issue="1", title="test", genres={"test", "test2", "test3", "test4"}, story_arcs=["arc1", "arc2"]
+    )
+    md.overlay(combine_cur, OverlayMode.combine_cur)
+
+    assert md == expected
```

Comment:
No, this is useless they can just reverse the call of `md.overlay(new_md)` to `new_md.overlay(md)`

Reply:
Again, because of the commandline clear?